### PR TITLE
fix(web): Try to not crash in unattended installation because of the storage section (#1739).

### PR DESCRIPTION
**Pull request by teclator**
## Problem

The **UI** is not ready for the new storage configuration loaded from the profile.

- https://github.com/agama-project/agama/issues/1739
- https://trello.com/c/3BVQ3zl5

## Solution

Adapted the installation finished page using the storage config instead of the proposal settings as it could not exist.

As the problem is reported just for unattended installation and the finished page we will provide a fix only for the reporeted issue as the whole UI should be ready when the new storage features [branch](https://github.com/agama-project/agama/tree/storage-config-ui) is merged 

## Testing

- *Tested manually*

### CODE REVIEW

File: web/src/api/storage/types/config.ts [hunk 1]
```diff
@@ -0,0 +1,369 @@
+/* eslint-disable */
+/**
+ * This file was automatically generated by json-schema-to-typescript.
```
Comment (imobachgs) [inline web/src/api/storage/types/config.ts:3]: We should decide on a tool to generate those files, but that's out of the scope of this PR.

File: web/src/queries/storage.ts [hunk 2]
```diff
@@ -123,6 +124,17 @@ const useDevices = (
...
 };
 
+const configQuery = {
+  queryKey: ["storage", "config"],
+  queryFn: fetchConfig,
+};
+
+const useConfig = () => {
+  const { data } = useSuspenseQuery(configQuery);
+  console.log(data);
```
Comment (imobachgs) [inline web/src/queries/storage.ts:134]: left-over

**Review by imobachgs — CHANGES_REQUESTED** [review]
It looks good. Please, update the changes file.

File: web/src/api/storage/types/config.ts (hunk 1 above)
Comment (teclator) [reply web/src/api/storage/types/config.ts:3]: Sure, BTW this types are fetched from the storage-config-ui branch (https://github.com/agama-project/agama/blob/storage-config-ui/web/src/api/storage/types/config.ts)

**Review by imobachgs — APPROVED** [review]

**Review by dgdavid — APPROVED** [review]

**Pull request merged**